### ISSUE TITLE
Add billboard test link to admin page

### ISSUE THREAD

**Issue by PhilipHow** [forem/forem#20350]
## Is this feature related to a problem? Please describe.

As a Forem Admin, I want to be able to test Billboards, so I can quickly see what they look like on a post.

## Describe the solution you’d like

Add a "Test" button to the Billboard edit page within the `/admin` area that appends the test parameters `bb_test_placement_area` and `bb_test_id` to a random (fixed or variable) post link and opens the link in a new window. The `bb_test_placement_area` should be correct to the currently selected placement area for the billboard.

## Definition of Done

- [ ] Add a "Test" button that appends the test parameters `bb_test_placement_area` and `bb_test_id` to a random (fixed or variable) post link and opens the link in a new window.
- [ ] The `bb_test_placement_area` should be correct to the currently selected placement area for the billboard.

## Additional Context

- This test view functionality was implemented in https://github.com/forem/forem/pull/20311

**Comment by PhilipHow** [comment]
@xao0isb yes you can - please make sure you've read the [contributors guide ](https://developers.forem.com/contributing-guide/forem)before beginning

**Comment by xao0isb** [comment]
If I understand everything correctly it seems that depending on the selected value of the replacement area a corresponding link should point either to the post or to the home page (those pages where billboard can appear). Also, the test link should not only be for checking the placement area but also for checking all changed information. But this is functionality for new issues I think

**Comment by PhilipHow** [comment]
@xao0isb I think you're right!

I'm happy to limit the scope of this task to whatever you're comfortable with.

**Comment by vishaldeepak** [comment]
@PhilipHow please assign this to me
